ethapi/api.go

```diff
@@ -1043,6 +1043,11 @@ func DoCall(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash
 		return nil, err
 	}
 	vmConfig := opera.DefaultVMConfig
+	// use EVM for transaction simulation
```

Comment:
Feature question: should we play it safe and only switch to geth if there is a code that is longer than 64KB? This way most requests would still be processed with the LFVM and only the special cases switch to geth.

In this case we would not even add a flag. It will just be the default, without alternative. Wdyt?